core/api/src/app/quiz/list.ts

```diff
@@ -0,0 +1,9 @@
+import { fillQuizInformation } from "@/domain/quiz"
+import { QuizRepository } from "@/services/mongoose"
+
+export const listQuizzesByAccountId = async (accountId: AccountId) => {
```

Comment:
please use 1 param rule (object)

Reply:
is it a new rule? afair, one of our pattern is that we can use non object where there is a single input to the method, we only force the use of object when there are multiple arguments

Reply:
the problem with this is tracing... if we dont use objects is difficult to trace and block leaks because param is mapped with a number (which is generic) while if you use an object we will have the name `code.function.params.paramName`